test/cljs/doit/core_test.cljs

```diff
@@ -30,7 +31,12 @@
          new-todo-done   {:content "new todo" :done true :id 3}
          all-todos       (rf/subscribe [::subs/todos])
          completed-todos (rf/subscribe [::subs/completed-todos])
-         remaining-todos (rf/subscribe [::subs/remaining-todos])]
+         remaining-todos (rf/subscribe [::subs/remaining-todos])
+         auth-token      (rf/subscribe [::subs/auth-token])]
+
+     (testing "signin success can set auth-token"
```

Comment:
Here, you are testing the `save-auth-token` handler and not the sign-in flow itself. Rename this to "Can set auth token" or something appropriate.